cmd/ethkit/wallet.go

```diff
@@ -13,8 +12,9 @@ import (
 	"github.com/0xsequence/ethkit/ethwallet"
 	"github.com/0xsequence/ethkit/go-ethereum/accounts/keystore"
 	"github.com/0xsequence/ethkit/go-ethereum/common"
+	"github.com/bytedance/sonic"
 	"github.com/spf13/cobra"
-	"golang.org/x/crypto/ssh/terminal"
```

Comment:
we want crypto/ssh/terminal .. its more secure

Reply:
From https://pkg.go.dev/golang.org/x/crypto/ssh/terminal

> Deprecated: this package moved to golang.org/x/term.